src/lsp-commands.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2019 Red Hat, Inc. and others.
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *     http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Disposable, commands } from 'vscode';
+
+namespace CommandKind {
+  export const COMMAND_REFERENCES = "quarkus.command.references";
+  export const COMMAND_IMPLEMENTATIONS = "quarkus.command.implementations";
+  export const COMMAND_OPEN_URI = "quarkus.command.open.uri";
+  export const COMMAND_CONFIGURATION_UPDATE = "quarkus.command.configuration.update";
+}
+
+export function registerConfigurationUpdateCommand(callback: (...args: any[]) => any): Disposable {
```

Comment:
remove callback parameter which should be implemented inside this registerConfigurationUpdateCommand function. If you need to add some parameter (like section, do it), but the idea is to copy/paste this file in an another vscode extension and it should work without changes (just change the `quarkus.command.configuration.update` into `xml.command.configuration.update` and that's all